packages/helm-charts/testnet/templates/forno.ingress.yaml

```diff
@@ -15,6 +15,7 @@ metadata:
     # Allows WS connections to be 20 minutes long, see https://kubernetes.github.io/ingress-nginx/user-guide/miscellaneous/#websockets
```

Comment:
Worth updating the apiVersion to networking.k8s.io/v1 if supported here as well?

Reply:
Considering we should be using k8s 1.20+ in all clusters, I have created this PR based on this recommendation.